autoload/wiki/init.vim

```diff
@@ -39,7 +39,7 @@ function! wiki#init#get_os() abort " {{{1
   if has('win32')
     return 'win'
   elseif has('unix')
-    if system('uname') =~# 'Darwin'
+    if has{'ios') || system('uname') =~# 'Darwin'
```

Comment:
It seems you've used `{` instead of `(` here. Also, perhaps we should also add `has('mac')`?